internal/service/service.go

```diff
@@ -137,3 +149,22 @@ func (s *Service) LoadAllRetrospectives(ctx context.Context) error {
 func (s *Service) GetLimits(ctx context.Context) *types.ApiLimits {
 	return types.GetApiLimits()
 }
+
+func (s *Service) CleanUpRetros(ctx context.Context) error {
+	config := config.Get()
+	cleanUpDays := time.Duration(config.Schedule.CleanUpDays)
+	date := time.Now().Add(-(cleanUpDays * 24 * time.Hour))
+	ids, err := s.repository.GetOldRetrospectives(ctx, date)
+	if err != nil {
+		return err
+	}
+
+	for _, id := range ids {
+		if _, err := s.repository.DeleteRetrospective(ctx, id); err != nil {
+			return err
+		}
+	}
+
+	log.Printf("deleted %d retrospectives older than %s", len(ids), date.String())
```

Comment:
maybe add a \n in the end of this print

Reply:
Not necessary, this function already break line after print